scripts/SREP/retry-pruning-cronjob/metadata.yaml

```diff
@@ -0,0 +1,35 @@
+file: script.sh
+name: fix-pruning-cron-job-error
+description: Restart the failed jobs in openshift-sre-pruning namespace
+author: Dee-6777
+allowedGroups:
+  - SREP
+labels:
+  - key: OSD_TYPES
+    description: Compatible cluster types for this script
+    values:
+      - OSD
+rbac:
+    roles:
+      - namespace: "openshift-sre-pruning"
+        rules:
+          - verbs:
+            - "get"
+            - "list"
+            - "watch"
+            - "delete"
+            apiGroups:
+            - "batch"
+            resources:
+            - "jobs"
+    clusterRoleRules:
```

Comment:
I am not sure if the clusterRoleRules are needed here.

Reply:
Thanks, I've removed that. We don't need permission for any cluster-scoped resources or across all namespaces.